web/src/main/java/com/futurewei/alcor/web/entity/dataplane/NetworkConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package com.futurewei.alcor.web.entity.dataplane;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.schema.Common;
+import com.futurewei.alcor.web.entity.securitygroup.SecurityGroupEntity;
+import com.futurewei.alcor.web.entity.subnet.SubnetEntity;
+import com.futurewei.alcor.web.entity.vpc.VpcEntity;
+import lombok.Data;
+
+@Data
+public class NetworkConfiguration {
+
+  private Common.ResourceType rsType;
```

Comment:
What are these? As commented in PR https://github.com/futurewei-cloud/alcor/pull/234/files and discussed offline, we don't need this in the north bound configuration.

Reply:
changed in dcede47